DeAceroDLabs/marketplace-frontend
# [DMP-754] Loading animation for button in ProductDetailView

## What is done

[[DMP-754] [Frontend] Agregar Loading al botón de agregar al carrito](https://deacero.atlassian.net/browse/DMP-754)

## How is done

1. Added a loading parameter to Button component
2. Instead of using children on loading, use the spinner html
3. When request for price comes back empty, return to disable

## How to test

1. Run the project via `npm start`
2. Head over to any product detail page
3. Select valid parameters and see how button changes when loading price of product

## Evidence
